Feature/projects

bettina-btt opened the pull request:
Finalizamos grande parte. Acreditamos que esteja pronta. Ficamos no aguardo.

bettina-btt committed:
in production

bettina-btt committed:
Componentizando

bettina-btt committed:
componente/button

Batutta33 committed:
teste

Batutta33 committed:
Merge branch 'feature/projects' of https://github.com/jldlxs/battuta into feature/filter-date

bettina-btt committed:
componente/button

Batutta33 committed:
Atualização

Batutta33 committed:
Merge branch 'feature/projects' of https://github.com/jldlxs/battuta into feature/filter-date

bettina-btt committed:
.

Batutta33 committed:
.

Batutta33 committed:
Merge branch 'feature/projects' of https://github.com/jldlxs/battuta into feature/filter-date

Batutta33 committed:
alteração

Batutta33 committed:
alteraçao de limpeza

Batutta33 committed:
alteração

Batutta33 committed:
Feature/filter date (#22)

bettina-btt committed:
commit 11/08/2025

bettina-btt committed:
Revert "Feature/filter date (#22)" (#23)

bettina-btt committed:
última atualização de layout

bettina-btt committed:
Merge pull request #24 from jldlxs/feature/filter-date

Feature/filter date

bettina-btt committed:
última atualização de layout

bettina-btt committed:
atualização

bettina-btt committed:
Merge branch 'feature/projects' into feature/filter-multiple-options

bettina-btt committed:
Merge pull request #25 from jldlxs/feature/filter-multiple-options

Feature/filter multiple options

bettina-btt committed:
upd

bettina-btt committed:
upd filter date

bettina-btt committed:
upd

bettina-btt committed:
upd

bettina-btt committed:
upd

bettina-btt committed:
upd

bettina-btt committed:
upd

bettina-btt committed:
organizando apenas

jldlxs reviewed (changes requested):
Exibir o "X" de remover filtro apenas quando houver filtro selecionado.
<img width="206" height="74" alt="image" src="https://github.com/user-attachments/assets/6ad88573-8a9f-42da-8dd5-23a4924f0daa" />

Exibir o botão "Limpar todos os filtros" apenas quando houver algum filtro selecionado.
E manter os filtros por datas na segunda linha dos filtros, independente do tamanho da tela. 
<img width="1575" height="162" alt="image" src="https://github.com/user-attachments/assets/64cd8119-2a41-4c3f-af6a-205972fc766d" />

Ajustar a rota para passar por /manager, ficando /manager/projects. Se não passar pelo /manager, o menu some.
<img width="529" height="798" alt="image" src="https://github.com/user-attachments/assets/aa62b5a4-3eb6-44e4-8a6c-1ec72cdca7e5" />

Ao fazer a busca por um CNPJ, ao apagar os últimos dígitos e digitar outro CNPJ, nenhum resultado aparece.
<img width="635" height="474" alt="image" src="https://github.com/user-attachments/assets/01033e21-61c9-4a62-b3d3-736ad690e339" />


Implementar mensagem quando não houver resultado para a busca, da mesma maneira que foi feito com a modal do menu de selecionar empresa.
<img width="635" height="474" alt="image" src="https://github.com/user-attachments/assets/7021ecfe-244c-453d-9745-4fde76c777db" />


Ajuste a cor do background da lista de projetos para ficar igual ao projeto do Figma.
Aplicar cursor-pointer aos projetos.

bettina-btt committed:
update. Nada sobre as alterações ainda.

Batutta33 committed:
alterações solicitadas realizadas

bettina-btt committed:
Update

Nenhuma mudança  significativa. Apenas organizando.

bettina-btt committed:
Erros arrumados.

Peço desculpas.
Não tinha recebido os erros e acabei não prestando a devida atenção.

jldlxs commented:
Eu estava conferindo os arquivos de design no Figma e percebi que a versão de vocês estava diferente da minha.
Não sei o que ocorreu com o arquivo de vocês, se foi algo durante a manipulação, mas estava diferente do arquivo original.
Atualizei novamente o arquivo Figma que vocês tem acesso com um cópia do arquivo original.
**Observem que ele possui um container cinza para o conteúdo principal. E que o background dos itens da lista de projetos é de fato branco.** Porém na análise anterior eu havia me baseado no arquivo de vocês. Peço que ajustem.

Padronizar os placeholders para **DD/MM/AAAA**.
<img width="475" height="180" alt="image" src="https://github.com/user-attachments/assets/1715fcf5-328a-46aa-9d5f-52be45b1ede6" />

Os placeholder devem ser inputs com máscara. Ou seja, ao digitar os números eles devem inserir as barras "/" automaticamente conforme o usuário digita.
Não deve ser campo de seleção de data, conforme está atualmente.
Na documentação do Ionic em ion-input tem informações. Se precisarem de ajude me informem.
<img width="665" height="477" alt="image" src="https://github.com/user-attachments/assets/d3a6acec-5a20-485a-a762-9dd9a12a1f9e" />

Quando não houver resultados, esconder as colunas da lista de resultado e exibir a mensagem: **Nenhum projeto encontrado.** Coloquei uma referência no Figma.
<img width="1331" height="287" alt="image" src="https://github.com/user-attachments/assets/ff9ca0c5-7869-4e12-818d-63952c18e6cf" />


O checkmark do filter-multiple-option estão diferentes do nosso padrão visual. Eles devem ficar iguais ao arquivo do Figma. Ou seja, iguais aos checkmarks usados em company-select.
Os checkmarks do filter-multiple-options estão com bordas mais grossas.
Vamos manter a consistência visual ao longo do projeto.
<img width="184" height="289" alt="image" src="https://github.com/user-attachments/assets/7447a0dc-4fb5-422d-9509-8024b29f6e91" />
<img width="184" height="287" alt="image" src="https://github.com/user-attachments/assets/2b204411-80de-47df-a56a-7860b6ed40f0" />


Não deixe os elementos quebrarem a linha internamente.  Mas eles podem quebrar a linha no sentido de alguns filtros irem para a linha de baixo quando esgotar o espaço horizontal da modal. Mas não quebrar a linha dentro da opção.
<img width="593" height="181" alt="image" src="https://github.com/user-attachments/assets/14939c0e-24f7-4759-8e40-ca8f993b84f5" />


Faça o recurso **Limpar todos os filtros** considerar o campo de busca também. Ou seja, ao digitar uma busca o botão deve ser exibido. E consequentemente ao ser clicado o campo de busca também deve ser resetado.
<img width="1336" height="166" alt="image" src="https://github.com/user-attachments/assets/adcc9a21-ef22-4457-97fb-cc863b801f66" />

bettina-btt committed:
Alterações Feitas

bettina-btt committed:
update

bettina-btt committed:
Erro ao abrir o Modal arrumado.

Batutta33 committed:
Alteração do filter date realizadas

Batutta33 committed:
atualização

bettina-btt committed:
upd

bettina-btt committed:
upd

bettina-btt commented:
Alterações Feitas nos filtros de data
